src/types.ts

```diff
@@ -50,21 +50,32 @@ export type EditAssetParams = {
   data: EditAssetData;
 };
 
+export type AssetMetadata = {
+  attributes?: { trait_type: string; value: string; display_type?: string }[];
+  description?: string;
+  external_url?: string;
+  image_url?: string;
+  name?: string;
+  org_image_url?: string;
+  original_id?: string;
+}
```

Comment:
Shouldn't name, image_url, be required if AssetMetadata type is used?

Reply:
Indeed 👍  Also made org_image_url and original_id required as it is in the docs